crates/bevy_ui/src/node_bundles.rs

```diff
@@ -326,3 +326,49 @@ impl Default for ButtonBundle {
         }
     }
 }
+
+/// A Ui Node that renders a shader
```

Comment:
Seems back to front, the node doesn't render the shader.
```suggestion
/// A UI node that is rendered using a [`UiMaterial`]
```